src/hyperlight_host/src/hypervisor/hyperlight_vm.rs

```diff
@@ -1080,3 +1111,911 @@ mod debug {
         }
     }
 }
+
+#[cfg(test)]
+#[cfg(feature = "init-paging")]
```

Comment:
this should be valid for more than just init paging feature?

Reply:
yes will fix before publishing pr

Reply:
Most of these tests do require 64bit mode. There is an integration test at least